testnet-node/README.md

```diff
@@ -0,0 +1,29 @@
+## Testnet Node
+
+This is the easiest way to get a Babylon fullnode up and running, ready for integration development.
+A node in the network is identified by it's public key and there can be multiple addresses associated to a public key.
```

Comment:
Some of this is really good, but some of it is a little too detailed.

I'd recommend replacing all of the public key big with this:
```
A node in the network is identified by its public key. A keystore containing a keypair is automatically generated on startup and stored inside a docker volume called `key-data`, protected by default with password `"password"`.
```

The ledger stuff is good :)

Reply:
I'd also suggest mentioning:
* `The node connects to the latest public testnet (as of April/May 2023, this is RCnet V1, known by its logical name kisharnet).`
* `[Documentation for integrators is available here](https://docs.google.com/document/d/1cjc7_alyzIb2QQIGGn1PEpJyjrMRZYHq3VwkOXRP8J0/edit#).`